app/macos/Podfile

```diff
@@ -38,8 +38,5 @@ end
 post_install do |installer|
   installer.pods_project.targets.each do |target|
     flutter_additional_macos_build_settings(target)
-    target.build_configurations.each do |bc|
-      bc.build_settings['MACOSX_DEPLOYMENT_TARGET'] = '10.14'
-    end
```

Comment:
This is not important?

Reply:
We needed this for M1 stuff back in the days. Also worked without it, so I would leave it for now. If a similar error occurs, we can add it again.